schematools/types.py

```diff
@@ -285,10 +333,14 @@ def sub_fields(self) -> typing.List[DatasetFieldSchema]:
         if not self.is_object:
             return
 
-        required = set(self["required"])
+        field_name_prefix = ""
+        if self.relation is not None:
+            field_name_prefix = self.relation.split(":")[1] + "__"
```

Comment:
Code is beetje magisch hier, misschien een docstring toegevoegd kan worden met korte omschrijving van waarom is tweede element gepakt?